docs/installation_and_tutorial.md

```diff
@@ -343,32 +260,43 @@ Finally the logs are shown in the runtime:
 
 ## Trends
 
-To add a trend go to the add screen items and in the advanced bar select trend control:
+To add a trend go to the Toolbox, select trend control and drag and drop into the screen:
 
-![trends1](graphics/trends1_new.jpg)
+![trends1](graphics/trends1.png)
 
-In its properties to add different trends to appear in the item, go to Trends:
+In its properties to add different trends to appear in the item, go to Trends and select the tag/logging tag you want to control::
 
 ![trends2](graphics/trends2.png)
 
-And select the tag you want to control:
-
-![trends3](graphics/trends3.png)
-
 In the runtime the trend will be filled:
 
 ![trends4](graphics/trends4.png)
 
-## Start the runtime
+## TIA Portal guidelines
+
+In case you need more information related to the engineering of the project in TIA Portal, you can refer to the following [documentation](https://cache.industry.siemens.com/dl/dl-media/433/109782433/att_1265883/v1/109782433_SIMATIC-WinCC-Unified-Tutorial-Center_140324_web/start.htm#!/en).
+
+## How to use WinCC Unified Runtime Manager
 
 To start the runtime go to the Wincc Runtime app:
 
 ![start1](graphics/start1.png)
 
+You will see a login page, where you can login with the credentials downloaded within the project. At least one user with administration rights is required to access the Web Runtime Manager.
+<br>Note: Default user: **uoeuser**, Default password: **User@uoe**
+
+![login](graphics/login.png)
+
+If you want to add a user to change the access rights or any other reason, you can do it in the TIA Project:
+
+![addUser](graphics/addUser.png)
+
 If the project downloaded is running you will see a green light that indicates that the runtime is ready:
 
 ![start2](graphics/start2.png)
 
+### Start the runtime
+
 Click on the WinCC Unified Runtime button:
 
 ![start3](graphics/start3.png)
```

Comment:
I would change this screenshot with a screenshot which shows more an application instead of a screen with some single items. We already have some nice screenshots, which fits better here. 
See following location:
https://siemens.sharepoint.com/:f:/r/teams/UnifiedEdgeG2MSync/Shared%20Documents/General/Info-Material/[Screenshots](https://siemens.sharepoint.com/:f:/r/teams/UnifiedEdgeG2MSync/Shared%20Documents/General/Info-Material/Screenshots?csf=1&web=1&e=cbkaLz)?csf=1&web=1&e=cbkaLz